api.proto

```diff
@@ -214,21 +214,33 @@ message StreamMetadata {
         OK             = 0;
         UNKNOWN_STREAM = 1;
     }
-    string                        name       = 1; // The name of the stream being described
-    string                        subject    = 2; // The stream subject
-    Error                         error      = 3; // Indicates if there was something wrong with the requested stream
-    map<int32, PartitionMetadata> partitions = 4; // Information for the stream partitions
+    string                        name              = 1; // The name of the stream being described
+    string                        subject           = 2; // The stream subject
+    Error                         error             = 3; // Indicates if there was something wrong with the requested stream
+    map<int32, PartitionMetadata> partitions        = 4; // Information for the stream partitions
+    int64                         creationTimestamp = 5 [jstype=JS_STRING]; // When the stream has been created
+}
+
+// PartitionEventMetrics contains metrics related to a stream partition.
+message PartitionEventMetrics {
```

Comment:
I wouldn't call these metrics, maybe call them `PartitionState` instead. With metric in the name people are at risk of confusing them with the proper metrics work

Reply:
I agree, but I'm not sure about a new name. I feel `State` implies the current state, so `pauseState` or `readonlyState`, but what's stored here is not really the state but rather some historical value about it. `PartitionEventHistory` seems a bit too much. `PartitionEventTimestamps`?

Reply:
yeah `PartitionEventTimestamps` sounds good

Reply:
Fixed in 143245f.